### PR TITLE
Build PNG and JPEG libraries.

### PR DESCRIPTION
**Pull request by aerorahul**
This PR builds PNG and JPEG libraries.
It also updates the README.md to reflect the changes (and some minor documentation consistencies).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,8 @@ The remaining items enable or disable builds of each software package.  The foll
 * Minimal JEDI Stack
   - SZip (Step 3)
   - Zlib
+  - PNG
+  - JPEG
```
Comment (mmiesch) [inline README.md:128]: The "Minimal JEDI Stack" is what is absolutely required to build JEDI.  PNG and JPEG are not absolutely required, are they?

File: buildscripts/config/config_cheyenne.sh [hunk 2]
```diff
@@ -32,7 +32,9 @@ export WGET="wget -nv"
 export      STACK_BUILD_CMAKE=N
 export       STACK_BUILD_SZIP=Y
 export    STACK_BUILD_UDUNITS=Y
+export       STACK_BUILD_JPEG=Y
 export       STACK_BUILD_ZLIB=Y
+export        STACK_BUILD_PNG=Y
```
Comment (mmiesch) [inline buildscripts/config/config_cheyenne.sh:37]: Again - shouldn't these be optional additions?

Comment (aerorahul) [reply buildscripts/config/config_cheyenne.sh:37]: yup, you are right.
We can turn them off by default.

File: README.md (hunk 1 above)
Comment (aerorahul) [reply README.md:128]: You are right.
These are puny libraries, that no one in JEDI is really is using right now. We can move them down to optional.

**Comment by mmiesch** [comment]
Other than that placement question everything looks good - it works on my Mac.

**Comment by aerorahul** [comment]
> Other than that placement question everything looks good - it works on my Mac.

thanks @mmiesch 
I'll make the changes and push and then merge.

**Comment by mmiesch** [comment]
Thanks @aerorahul - I also checked on AWS/ubuntu with intel 19 and it works there too.

**Comment by mmiesch** [comment]
I meant not only to set them to N but also to move them to the optional section on the bottom.  But - doesn't matter too much

**Pull request merged**